template.yaml

```diff
@@ -230,6 +236,74 @@ Resources:
         - Key: Owner
           Value: di-orchestration@digital.cabinet-office.gov.uk
 
+  PerfTestContainerService:
+    Type: AWS::ECS::Service
+    Condition: IsStaging
+    Properties:
+      Cluster: !Ref FargateCluster
+      TaskDefinition: !Ref PerfTestTaskDefinition
+      DesiredCount: 1
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: DISABLED
+          Subnets:
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdA
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdB
+          SecurityGroups:
+            - !Ref PerfTestContainerServiceSecurityGroup
+      LoadBalancers:
+        - ContainerName: !Sub ${AWS::StackName}-PerfTestECSServiceTaskDefinition
+          ContainerPort: 8080
+          TargetGroupArn: !Ref PerfTestApplicationLoadBalancerTargetGroup
+      PropagateTags: SERVICE
+      Tags:
+        - Key: Name
+          Value: !Sub ${AWS::StackName}-PerfTestContainerService
+        - Key: Service
+          Value: Relying Party Performance Testing stub
+        - Key: Source
+          Value: alphagov/di-auth-stub-relying-party
+        - Key: Owner
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+    DependsOn: PerfTestApplicationLoadBalancerListener
+
+  PerfTestContainerServiceSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Condition: IsStaging
+    Properties:
+      GroupDescription: Security Group to access the Container Service
+      GroupName: !Join
+        - "-"
+        - - !Ref AWS::StackName
+          - PerfTestContainerService
+          - Fn::Select:
+              - 4
+              - Fn::Split:
+                  - "-"
+                  - Fn::Select:
+                      - 2
+                      - Fn::Split:
+                          - "/"
+                          - Ref: AWS::StackId
+      SecurityGroupIngress:
+        - Description: Allow traffic from the load balancer on port 8080
+          SourceSecurityGroupId: !GetAtt PerfTestApplicationLoadBalancerSecurityGroup.GroupId
+          IpProtocol: tcp
+          FromPort: 8080
+          ToPort: 8080
+      VpcId:
+        Fn::ImportValue: !Sub ${VpcStackName}-VpcId
+      Tags:
+        - Key: Name
+          Value: !Sub ${AWS::StackName}-PerfTestContainerServiceSecurityGroup
+        - Key: Service
+          Value: Relying Party Performance Testing stub
+        - Key: Source
+          Value: alphagov/di-auth-stub-relying-party
```

Comment:
one login org

Reply:
happy for you to change all of these in a new PR

Reply:
ah I've now done the ones that aren't the perf test stub, so I should probably update this now. Thanks!